osclib/obslock.py

```diff
@@ -74,7 +74,7 @@ def _read(self):
         return signature
 
     def _write(self, signature):
-        url = makeurl(self.apiurl, ['source', self.lock, '_attribute', '%s:LockedBy' % self.ns])
+        url = makeurl(self.apiurl, ['source', self.lock, '_attribute'])
```

Comment:
- should be mirrored in `_read()`?
- should likely be removed from config if no longer used

Reply:
No, in read it works as filter. In write it's just ignored by OBS and it only takes what's in XML. The API for attributes is most likely the worst in OBS.